### PR TITLE
Delete unused test case and rename func name

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
Delete unused test case and rename func name
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @liggitt
thanks

File: pkg/credentialprovider/aws/aws_credentials_test.go [hunk 1]
```diff
@@ -108,56 +108,3 @@ func TestEcrProvide(t *testing.T) {
 		}
 	}
 }
-
-func TestChinaEcrProvide(t *testing.T) {
```
Comment (liggitt) [inline pkg/credentialprovider/aws/aws_credentials_test.go]: in what way was this unused?

Comment (jianglingxia) [reply pkg/credentialprovider/aws/aws_credentials_test.go]: the TestEcrProvide func has tested,only the para "cn-foo-1" and "lala-land-1" deffierent,I don't know if I understand wrong,thanks

**Comment by justinsb** [comment]
Thanks for the contribution @jianglingxia 

The tests do test different code paths - the second test is testing the special logic around the AWS CN (china) region.  I agree the tests are repetitive and could be parameterized.

If you wanted to use the "testgrid pattern" to remove the code repetition, that would be great, otherwise I suggest we close the PR.

testgrid pattern example: https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/tags_test.go#L82

**Pull request closed without merging**